compliance/us_core.rst

```diff
@@ -0,0 +1,66 @@
+US CORE
+=======
+
+* Tested Version: Firely Server has been tested against `US Core Version 4.0.0 - STU4 Release <https://hl7.org/fhir/us/core/STU4/terminology.html>`_
+* All StructureDefinitions for profiles and extensions are loaded by default in the standard SQlite Administration Database of Firely Server. No additional configuration needed in order to validate against these conformance resources.
+* Firely Server provides full `"Profile Support and Interaction Support" <http://hl7.org/fhir/us/core/STU4/conformance-expectations.html#profile-support--interaction-support>`_:
+  
+  * Firely Server can be populated with resources conforming to US Core, including all elements marked as Must-Support
+  * All search and CRUD interactions defined by US Core are supported, including optional search parameters
+  
+See `How to Test Firely Server on Inferno <https://fire.ly/ebook-how-to-test-firely-server-on-inferno/>`_ for more information on how to pass the offical 21st Century Cures Act test.
+  
+Known Limitations
+^^^^^^^^^^^^^^^^^
+
+* In order to validate resources claiming to conform to US Core, it is necessary to configure Firely Server to use an external terminology server incl. support for expanding SNOMED CT and LOINC ValueSets. See :ref:`feature_terminology`.
+* The $docref operation is not yet supported on DocumentReference resources
```

Comment:
```suggestion
* The ``$docref`` operation is not yet supported on DocumentReference resources
```
Suggestion: formatting the operation as code. A matter of taste, so you can ignore this if you want. :-)